tests/data/dlc_multiple_datasets/video2/dlc_dataset_2.csv

```diff
@@ -0,0 +1,7 @@
+scorer,Scorer,Scorer,Scorer,Scorer,Scorer,Scorer,Scorer,Scorer,Scorer,Scorer,Scorer,Scorer
+individuals,Animal1,Animal1,Animal1,Animal1,Animal1,Animal1,Animal2,Animal2,Animal2,Animal2,Animal2,Animal2
+bodyparts,A,A,B,B,C,C,A,A,B,B,C,C
+coords,x,y,x,y,x,y,x,y,x,y,x,y
+labeled-data/video/img000.png,0,1,2,3,4,5,6,7,8,9,10,11
+labeled-data/video/img001.png,12,13,,,15,16,17,18,,,20,21
+labeled-data/video/img002.png,22,23,24,25,26,27,,,,,,
```

Comment:
Maybe change a couple of these numbers so we can test that we're not just importing the same thing twice?

Reply:
Split the 2 files to have a bit more realistic test. One has labels from the first 2 frames, the other one from the last frame. Also added the images for manual verification via UI